### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by fwittenfeld** [LarsK1/hass_solvis_control#2] [General]
Erstmal vielen Dank für die coole Integration. Klappt bei mir direkt ziemlich gut.

Bei mir werden leider zwei Werte negativ angezeigt: # Heizungspuffertemperatur unten, # Brennerstarts
Diese hast du auch mit `negative=True` hinterlegt. War das wirklich beabsichtigt?

Was hat es mit der Ziffer bei dieser Bezeichnung auf sich? `WW Nachheizung 2322`

Bei den Bezeichnungen finde ich ehrlich gesagt die Variante aus diesem Projekt übersichtlicher (Kesselwerte benachbart etc):
https://github.com/Patrick762/hassio-solvis-modbus
Kennst du das Projekt? Ihr seid offenbar fast gleichzeitig gestartet.
Vielleicht kann man da Anstrengungen bündeln oder sich gegenseitig zumindest inspirieren? ^^
![Patrick762_Solvis](https://github.com/LarsK1/hass_solvis_control/assets/29568699/c9a5303b-7f01-4412-a534-0cb1bfc9e7fe)

Wenn du magst, könnte ich ggf. auch PullRequests stellen. Allerdings macht es dafür Sinn, wenn du zunächst eine Lizenz für dein Projekt hinterlegst.

**Comment by LarsK1** [comment]
Hallo, 
vielen Dank für deinen Input. Die Lizenz habe ich bereits hinzugefügt. Die beiden Werte schaue ich mir morgen noch einmal an. 

Die"skurrile" Bezeichnung ist die Modbus-Adresse - diese Bibliothek ist ja auch aus vielen Blogeinträgen entstanden...

Das andere Projekt kannte ich bisher noch nicht.

Wenn du Änderungen hast oder beitragen möchtest/ kannst sehr sehr gerne!

**Reply by Patrick762** [reply]
Interessant dass hier auch an einer Integration gearbeitet wurde. SEO muss ich wohl noch lernen. Dann hättest du die Integration wohl gefunden

**Comment by LarsK1** [comment]
Ich habe mir gerade noch einmal die beiden Felder angeschaut:
![image](https://github.com/LarsK1/hass_solvis_control/assets/28981528/992f6d3e-fd4c-4f87-be53-70bef2032f5b)
![image](https://github.com/LarsK1/hass_solvis_control/assets/28981528/0158fabc-879a-43ab-aff3-cc66d608d739)
Bei mir sind aber beide positiv. Wenn ich gerade auch selbst irritiert bin, warum die Anzahl der Brennerstarts stetig abnimmt.

Welche Softwareversion der SC3 betreibst du den?

**Reply by fwittenfeld** [reply]
Meine SC3 hat die Version MA3.18.14
Ich habe in HA Brennerstarts -1.862, an der Heizung +1862
Einen hübschen Plot habe ich hier nicht, da der Brenner derzeit aus ist. ;-)

Meine Puffertemperatur schaut plausibel aus, jedoch mit negativem Vorzeichen.
![PufferTunten](https://github.com/LarsK1/hass_solvis_control/assets/29568699/e0f7b717-3aaf-4710-9143-35b9ba8f43cd)

Zum Vergleich
![Puffer](https://github.com/LarsK1/hass_solvis_control/assets/29568699/ae931ec1-e540-4ed7-938e-eba5e6525997)

**Reply by LarsK1** [reply]
Ich habe mal einen Pre-Release gemacht. Gerne einmal updaten und Bescheid geben.

**Reply by fwittenfeld** [reply]
Die beiden Vorzeichenfehler sind bei mir gefixt!
Danke für deinen fixen Support! =)
